swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
+
 class Item:
-    pass
\ No newline at end of file
+    category = ""
```

Comment:
Assigning variables in the class like this is actually making a class variable, not an instance variable. We only needed instance variables in this project.